### DISCUSSION TITLE
PorousFlow - Strain

### DISCUSSION THREAD

**Discussion by MatiasAllay** [idaholab/moose#18290] [Q&A Modules: General] [answered]
Hi,
Why my RZ PorousFlow model converges nicely when the strain is computed with `ComputeSmallStrain`, but it does not with `ComputeAxisymmetricRZSmallStrain`? Is it correct to use `ComputeSmallStrain` with cylindrical coordinates? If not, how can I help MOOSE to converge using `ComputeAxisymmetricRZSmallStrain`?
Thanks again!
Matias

**Comment by WilkAndy** [comment, marked as answer]
You must use `ComputeAxisymmetricRZSmallStrain` .   You've possibly omitted another "RZ" thing, or maybe your BCs are wrong.   Inspect the test files that have "RZ" in them and compare with yours.  I'd start with
```
.test/tests/thm_rehbinder/fixed_outer_rz.i
examples/tutorial/11_2D. i
```

a

**Reply by MatiasAllay** [reply]
Thanks once again!
